src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -413,34 +412,67 @@ def _alloc_from_dict(self, config_dict):
                     "Loading GEN_KW from files requires %d in file format"
                 )
 
+        self.installed_jobs = self._installed_jobs_from_dict(config_dict)
         jobs = []
         # FORWARD_MODEL_KEY
         for job_description in config_dict.get(ConfigKeys.FORWARD_MODEL, []):
-            job = self.site_config.job_list.get_job_copy(
-                job_description[ConfigKeys.NAME]
-            )
+            job = self.installed_jobs[ConfigKeys.NAME].copy()
             job.set_private_args_as_string(job_description.get(ConfigKeys.ARGLIST))
             job.convertToCReference(None)
             jobs.append(job)
 
         # SIMULATION_JOB_KEY
         for job_description in config_dict.get(ConfigKeys.SIMULATION_JOB, []):
-            job = self.site_config.job_list.get_job_copy(
-                job_description[ConfigKeys.NAME]
-            )
+            job = self.installed_jobs[ConfigKeys.NAME].copy()
```

Comment:
I think this needs to be:
  job = self.installed_jobs[job_description[ConfigKeys.NAME]].copy()